evap/evaluation/models.py

```diff
@@ -522,26 +535,34 @@ class Meta:
         verbose_name_plural = _("answers")
 
 
-class LikertAnswer(Answer):
+class LikertAnswerCounter(Answer):
     """A Likert-scale answer to a question with `1` being *strongly agree* and `5`
     being *strongly disagree*."""
 
     answer = models.IntegerField(verbose_name=_("answer"))
+    count = models.IntegerField(verbose_name=_("count"), default=0)
 
     class Meta:
         verbose_name = _("Likert answer")
         verbose_name_plural = _("Likert answers")
```

Comment:
could you try ensuring uniqueness of an answer value for one question? like, there should be no two answercounters with the same answer for the same question. should be possible somehow with the "unique" attribute of the meta class.